_includes/tools/ava/usage.md

```diff
@@ -0,0 +1,40 @@
+AVA comes with built-in support for Babel 6 and let you write your tests
```

Comment:
Maybe emphasize more that AVA doesn't require extra setup to use babel with es2015, but if you want you can customize it? cc @sindresorhus